flows/engine/engine.go

```diff
@@ -95,20 +97,6 @@ func ResumeFlow(env flows.FlowEnvironment, run flows.FlowRun, event flows.Event)
 	return run.Session(), nil
 }
 
-// initializes our context based on our flow and current context
-func initTranslations(run flows.FlowRun) {
-	// set our language based on our contact if we have one
-	contact := run.Contact()
-	if contact != nil {
```

Comment:
What is the usecase for a run without a contact? If there's a good usecase then it's worth it. Otherwise it's a nuisance to have to keep checking `run.Contact() != nil` and from a glance it looks like we don't always do that, e.g. https://github.com/nyaruka/goflow/blob/master/flows/engine/engine.go#L14